tests/tests/test-txpool.ts

```diff
@@ -0,0 +1,104 @@
+import { expect } from "chai";
+
+import { createAndFinalizeBlock, customRequest, describeWithMoonbeam } from "./util";
+import { ERC20_BYTECODE } from "./constants/testContracts";
+import { GENESIS_ACCOUNT, GENESIS_ACCOUNT_PRIVATE_KEY } from "./constants";
+
+let txContract, contractAddress;
+
+describeWithMoonbeam("Moonbeam RPC (TxPool RPC module)", `simple-specs.json`, (context) => {
```

Comment:
There is no test to verify also the tx content (`"0x{:x}: {} wei + {} gas x {} wei",`) ?

Reply:
Thanks for taking some time to look at it. Do you mean testing the `txpool_inspect` response? That should be covered by [this](https://github.com/PureStake/moonbeam/pull/171/files/9ef0bf4fd031ee27b77189adc43cde7a52177982#diff-8c64edf466f2b72733aaf111f533bb19520255689ffae126a995b3e44ca1cebdR28) and [this](https://github.com/PureStake/moonbeam/pull/171/files/9ef0bf4fd031ee27b77189adc43cde7a52177982#diff-8c64edf466f2b72733aaf111f533bb19520255689ffae126a995b3e44ca1cebdR86).

Reply:
Haha sorry, I was a bit tired and missed those lines :)